### ISSUE TITLE
v1.1.0 npm release

### ISSUE THREAD

**Issue by HIMISOCOOL** [KoRiGaN/Vue2Leaflet#235]
Hey there team, any eta for a new NPM release? I just realized 1.0.2 doesn't have LControl exported :(

**Comment by lordfuoco** [comment]
@HIMISOCOOL there is a problem with the travis npm deploy that sadly I can't solve alone waiting for @KoRiGaN input

**Comment by lordfuoco** [comment]
@HIMISOCOOL while the npm situation is blocked I've made a next branch where I will keep the most updated code available under the dist folder, you can add it to your repo with:
`"vue2-leaflet": "git+https://github.com/KoRiGaN/Vue2Leaflet.git#next"`
Just be aware that is always going to be bleeding edge code :)

**Comment by HIMISOCOOL** [comment]
Thanks heaps man, Im a bloody sort of dude anyway XD

**Comment by lordfuoco** [comment]
@HIMISOCOOL :D if you find any bugs / problems please let me know here

**Comment by HIMISOCOOL** [comment]
@lordfuoco When I try to make a CircleMarker like in examples/src/components/GeometryTest.vue
it fails to initialize saying:

> TypeError: Cannot read property 'x' of undefined
>     at Bounds.intersects (leaflet-src.js?2002:1053)
>     at NewClass._empty (leaflet-src.js?e11e:7899)
>     at NewClass._updateCircle (leaflet-src.js?2002:12627)
>     at NewClass._updatePath (leaflet-src.js?e11e:7895)
>     at NewClass._update (leaflet-src.js?e11e:7890)
>     at NewClass._reset (leaflet-src.js?e11e:7806)
>     at NewClass.onAdd (leaflet-src.js?e11e:7754)
>     at NewClass._layerAdd (leaflet-src.js?e11e:6531)
>     at NewClass.whenReady (leaflet-src.js?2002:4367)
>     at NewClass.addLayer (leaflet-src.js?2002:6549)

**Comment by HIMISOCOOL** [comment]
well, I accidentally had lat and long around the wrong way so it was complaining about calculating bounds for a circlemarker earlier today but no mention of lat and long resolving to unusable co-ordinates so perhaps an improvement there? Maybe a validator that tells you your values are way too big.

My popups for markers were not showing at all on 1.1.0
Otherwise the all the other vue wrappers will need updating too, I unfortunately had to rollback to 1.0.2 because of google mutant wrapper being mad.

**Comment by lordfuoco** [comment]
@HIMISOCOOL so to capture your feedback:

- You would like a warning when lat and lon are passed improperly
- Popup issue (aware of it should be fixed now ) 
- Vue wrappers need updating ? this I don't understand

Thank you anyhow this is really helpful

**Comment by HIMISOCOOL** [comment]
https://github.com/jperelli/vue2-leaflet-googlemutant 1.0.2 throws a very large number of errors when interacting with the map when using Vue2Leaflet 1.1.0.
It renders fine so I don't know if it affects its actual usage but its not a good time for the console

**Comment by lordfuoco** [comment]
@HIMISOCOOL there was a bug with the default overlay pane for marker and gridLayer, I fixed that in the next.1 version, if you have time and you could try if this works for you now it would be great.
Also tooltip are working back again. 
What kind of tooltip are you using ?

**Comment by HIMISOCOOL** [comment]
```
<l-marker
    v-if="!!selectedObject"
    :draggable="false"
    :lat-lng="objectLocation"
    :icon="objectIcon"
>
    <l-popup :content="objectName"></l-popup>
</l-marker>
```

if that's what you mean by tooltip

**Comment by lordfuoco** [comment]
@HIMISOCOOL Yes sorry, I should have asked if you were using a pop / tooltip with a slot content or with content as a prop (they differ in implementation ) 
The implementation with a prop was indeed broken at some point, is now working properly (be sure to be running next.4 ) 
Apologise for the instability of the branch, now it will be better

**Comment by HIMISOCOOL** [comment]
its okay dude, I just had to commit working code quickly, If I had my way
it would be done when its done XD
I will let you know how everything goes hopefully tomorrow

**Comment by lordfuoco** [comment]
@HIMISOCOOL Same here :) thanks a lot
